spec/spec_helper.rb

```diff
@@ -8,6 +8,10 @@
 
 require "manageiq-providers-openstack"
 
+RSpec.configure do |config|
+  config.filter_run_excluding(:qpid_proton) unless Gem.loaded_specs.key?(:qpid_proton)
```

Comment:
I think we need to at least keep the check for CI env car.  While locally I don't mind if the specs are skipped, we should never skip them on CI.

Reply:
@Fryguy ok, added.